sessionctx/variable/session.go

```diff
@@ -336,14 +336,14 @@ type TableDelta struct {
 type StatementContext struct {
 	// Set the following variables before execution
 
-	InInsertStmt         bool
-	InUpdateOrDeleteStmt bool
-	InSelectStmt         bool
-	IgnoreTruncate       bool
-	IgnoreError          bool
-	TruncateAsWarning    bool
-	OverflowAsWarning    bool
-	InShowWarning        bool
+	InInsertStmt           bool
+	InUpdateOrDeleteStmt   bool
+	InSelectStmt           bool
+	IgnoreTruncate         bool
+	DividedByZeroAsWarning bool
```

Comment:
Do we need an `IgnoreDiviedByZero` ?

Reply:
I  haven't found that we can ignore `divide by zero` error when do the mod operation(according to MySQL documents). I think we can add it if we meet that situation in future.